pkg/cmd/admin/admin.go

```diff
@@ -22,7 +22,8 @@ import (
 	"github.com/openshift/origin/pkg/version"
 )
 
-const adminLong = `OpenShift Administrative Commands
+const adminLong = `
+OpenShift Administrative Commands
```

Comment:
Why add the newline that's just going to get trimmed later on?  Also, why force someone making a template to know that they need to trim?